tools/requirements_vendor.txt

```diff
@@ -1,23 +1,55 @@
+# This file was autogenerated by uv via the following command:
+#    uv pip compile --universal --python-version 3.8 requirements_vendor.in -o requirements_vendor.txt
+
+# This file was then modified to remove all markers, as we want to make
```

Comment:
I'm not a big fan of this having undefined post-processing steps. Is it something you could write a short script for?

Reply:
Done!